src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double base;
+    private double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    public RightTriangle(String color) {
+        super(color);
+    }
```

Comment:
It seems like the second constructor `RightTriangle(String color)` is not being used to set `base` and `height`. If you're not using this constructor to create a RightTriangle with default or calculated values for `base` and `height`, consider removing it or implementing it correctly to avoid having incomplete objects.